cmd/testbucket/ingest.go

```
@@ -0,0 +1,521 @@
+	for _, v := range in[1:] {
+		if v != out[len(out)-1] {
+			out = append(out, v)
+		}
+	}
+	return out
+}
+
+func (r *ingestReport) write(out io.Writer, prefix string) error {
+	ew := &errWriter{w: out}
+	w := io.Writer(ew)
+	fmt.Fprintf(w, "testbucket ingest — %d events (%d unparsable lines), alpha=%.2f\n", r.Events, r.Malformed, r.Alpha)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)